monitoring/monitors/alchemy/main.js

```diff
@@ -0,0 +1,232 @@
+      tokens: [],
+    },
+    token: {
+      name: 'Token',
+      address: AUDIUS_CONTRACT,
+      tokens: [],
+    },
+    governance: {
+      name: 'Governance',
+      address: '0x4DEcA517D6817B6510798b7328F2314d3003AbAC',
+      tokens: [],
+    },
+    staking: {
+      name: 'Staking',
+      address: '0xe6D97B2099F142513be7A2a068bE040656Ae4591',
+      tokens: [],
+    },
+    delegateManager: {
+      name: 'DelegateManager',
+      address: '0x4d7968ebfD390D5E7926Cb3587C39eFf2F9FB225',
+      tokens: [],
+    },
+    claimsManager: {
+      name: 'ClaimsManager',
+      address: '0x44617F9dCEd9787C3B06a05B35B4C779a2AA1334',
+      tokens: [],
+    },
+    serviceTypeManager: {
+      name: 'ServiceTypeManager',
+      address: '0x9EfB0f4F38aFbb4b0984D00C126E97E21b8417C5',
+      tokens: [],
+    },
+    serviceProviderFactory: {
+      name: 'ServiceProviderFactory',
+      address: '0xD17A9bc90c582249e211a4f4b16721e7f65156c8',
+      tokens: [],
+    },
+    ethRewardsManager: {
+      name: 'EthRewardsManager',
+      address: '0x5aa6B99A2B461bA8E97207740f0A689C5C39C3b0',
+      tokens: [],
+    },
+    wormholeClient: {
+      name: 'WormholeClient',
+      address: '0x6E7a1F7339bbB62b23D44797b63e4258d283E095',
+      tokens: [],
+    },
+    trustedNotifierManager: {
+      name: 'TrustedNotifierManager',
+      address: '0x6f08105c8CEef2BC5653640fcdbBE1e7bb519D39',
+      tokens: [],
+    },
+    // ClaimableTokens: {
+    //   address: 'Ewkv3JahEFRKkcJmpoKB7pXbnUHwjAyXiwEo4ZY2rezQ',
+    //   tokens: [],
+    // },
+    // RewardsManager: {
+    //   address: 'DDZDcYdQFEMwcu2Mwo75yGFjJ1mUQyyXLWzhZLEVFcei',
+    //   tokens: [],
+    // },
+    multiSig: {
+      name: 'MultiSig',
+      address: '0xeABCcd75dA6c021c7553dB4A74CACC958812432A',
+      tokens: [],
+    },
+  }
+
+  for (const key in addresses) {
+    const address = addresses[key].address
+
+    // Get token balances
+    const balances = await web3.alchemy.getTokenBalances(address)
+
+    // Remove tokens with zero balance
+    const nonZeroBalances = balances.tokenBalances.filter(token => {
+      return token.tokenBalance !== '0'
+    })
+
+    // Minimize variable churn
+    let name
+    let symbol
+    let decimals
+
+    // Loop through all tokens with non-zero balance
+    for (const token of nonZeroBalances) {
+      // Get balance of token
+      let balance = token.tokenBalance
+
+      if (token.contractAddress === AUDIUS_CONTRACT) {
+        // Skip API call for common request
+        name = 'Audius'
+        decimals = AUDIUS_DECIMALS
+        symbol = 'AUDIO'
+      } else {
+        // Get metadata of non-AUDIO token
+        const metadata = await web3.alchemy.getTokenMetadata(token.contractAddress)
+        name = metadata.name
+        decimals = metadata.decimals
+        symbol = metadata.symbol
+      }
+
+      // Compute token balance in human-readable format
+      balance = balance / Math.pow(10, decimals)
```

Comment:
i'm concerned this might lead to integer overflow in javascript. we use BN.js to do operations on numbers this large. web3 literally comes with this baked in to do token calculations https://web3js.readthedocs.io/en/v1.2.11/web3-utils.html#bn